skimage/transform/hough_transform.py

```diff
@@ -257,43 +153,30 @@ def hough_circle(image, radius, normalize=True, full_output=False):
         Hough transform accumulator for each radius.
         R designates the larger radius if full_output is True.
         Otherwise, R = 0.
-
-    Examples
```

Comment:
I agree to remove this for now - is a very complicated example. Any plans to add something instead in the near future?